### PR TITLE
Added Prefix Sum in C

### LINKED ISSUE #849: Prefix_sum in C
Hey this is Dipanwita. I am an RGSoC participant this year. I'd like to implement prefix sum in C.

### PR DESCRIPTION
**Pull request by susiejojo**
Fixes #849

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added Prefix_Sum.c

#### Languages Used:
- C

#### Files Added:
- Prefix_Sum.c

### CODE REVIEW

File: Prefix_Sum/Prefix_Sum.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#define MAX 1000
+
+int prefixsum[MAX];//array to store prefixsums
+
+/*function that calculates prefix sum by 
+forming the prefixsum array
+*/
+void calculate_prefix_sum (int arr[], int n)
+{
+	prefixsum[0] = arr[0]; //starts with the first element
```
Comment (jainaman224) [inline Prefix_Sum/Prefix_Sum.c]: Please use 4 space indentation throughout the file.

File: Prefix_Sum/Prefix_Sum.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{
+	int arr[] = { 1, 2, 3, 4, 5};
+	int n = sizeof(arr) / sizeof(arr[0]);
+	calculate_prefix_sum(arr, n);
+	for (int i = 0; i < n; i++)
+		printf ("%d ",prefixsum[i]);
+	printf("\n");
+}
+/* OUTPUT
+1 3 6 10 15 */
```
Comment (jainaman224) [inline Prefix_Sum/Prefix_Sum.c]: Add a single new line at the end of file.

**Comment by susiejojo** [comment]
Commit fa06829

File: Prefix_Sum/Prefix_Sum.c (hunk 1 above)
Comment (susiejojo) [reply Prefix_Sum/Prefix_Sum.c]: pls check commit fa06829

File: Prefix_Sum/Prefix_Sum.c (hunk 2 above)
Comment (susiejojo) [reply Prefix_Sum/Prefix_Sum.c]: pls check commit fa06829

**Comment by jainaman224** [comment]
👍

**Pull request merged**